### ISSUE TITLE
error: "st_extent" is an aggregate function

### ISSUE THREAD

**Issue by atonse** [michaelsogos/pg-diff#11]
Version: 1.2.8
PG Version: 11.2
Plugins: PostGIS is installed

When comparing two databases and PostGIS is installed, this seems to blow up with the following error:

```js
error: "st_extent" is an aggregate function
    at Connection.parseE (/Users/ashish/.nvm/versions/node/v10.15.1/lib/node_modules/pg-diff-cli/node_modules/pg/lib/connection.js:602:11)
    at Connection.parseMessage (/Users/ashish/.nvm/versions/node/v10.15.1/lib/node_modules/pg-diff-cli/node_modules/pg/lib/connection.js:399:19)
    at Socket.<anonymous> (/Users/ashish/.nvm/versions/node/v10.15.1/lib/node_modules/pg-diff-cli/node_modules/pg/lib/connection.js:121:22)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
```

**Comment by michaelsogos** [comment]
@atonse 

We will investigate connection problems with PostGIS, for now it is not supported.
From my understanding seems a problem with node-postgres library.

Happy new year 😄

**Comment by atonse** [comment]
@michaelsogos PostGIS is a plugin for Postgres, so we are actually just running vanilla Postgres. It's likely that to support PostGIS (not asking for support of PostGIS necessarily, because I mostly want my PG tables compared), you'd have to ignore `st_*` functions and it would already work.

However as I write this, I realize if you are doing data diffing, you'll initially want to ignore geometry tables. So maybe not so easy :-)

**Comment by michaelsogos** [comment]
Hi @atonse 
PostGIS is one of the most famous extension/distro for Postgre.
It is our intention to officially support it. 

We are working on it and before proceed with a simple "ignore st_* table" we would like to investigate deeply.

Thanks to support our library, we appreciate it.

**Comment by michaelsogos** [comment]
Hi @atonse ,

We released verison 1.2.13 that should resolve your issue.
Can you eventually let me know if it works for you?

Changelog:
- Updated to pg 8.1.0
- Exlusion of object created by extensions
- Support for any data type declared as USER DEFINED (like geometry 😄)
- Support POSGIS and PG_CRYPTO also in data compare
